astroid/rebuilder.py

```diff
@@ -451,6 +456,9 @@ def _save_assignment(self, node: nodes.AssignName | nodes.DelName) -> None:
         """Save assignment situation since node.parent is not available yet."""
```

Comment:
By the way, I'm a bit confused by this comment. What does it mean by `node.parent is not available`?

Reply:
I don't think so. This should be okay.